linode/manager
# refactor: [M3-6272] - MUI v5 Migration - `SRC > Features > Linodes` pt2

## Description 📝
Part 2 of migrating the Linodes folder to MUI v5

## Major Changes 🔄
- made exports named (unless lazily imported)
- cleanup - remove React.FC, rename props to be more specific (only if they were being exported)
- converted some class components to functional components (note -- some of the more complicated ones like CreateLinodes.tsx left as is)
- removed unnecessary index files / updated respective imports

## How to test 🧪
1. Go to the Linodes pages and make sure functionality works as expected and that styling looks the same
